src/main/scala/tilelink/Filter.scala

```diff
@@ -77,10 +77,23 @@ object TLFilter
   def mIdentity: ManagerFilter = { m => Some(m) }
   // preserve client visibility
   def cIdentity: ClientFilter = { c => Some(c) }
+
   // make only the intersected address sets visible
   def mSelectIntersect(select: AddressSet): ManagerFilter = { m =>
     val filtered = m.address.map(_.intersect(select)).flatten
     val alignment = select.alignment /* alignment 0 means 'select' selected everything */
+    intersectionHelper(m, filtered, alignment)
+  }
+
+  // make everything except the intersected address sets visible
+  def mExceptIntersect(except: AddressSet): ManagerFilter = { m =>
+    val filtered = m.address.flatMap(_.subtract(except))
+    val alignment: BigInt = if (filtered.isEmpty) 0 else filtered.map(_.alignment).max
```

Comment:
I think this should be 'min', not 'max'. You can't have a transfer larger than the min alignment of any AddressSet.